modern/src/settings/GroupsPage.js

```diff
@@ -45,13 +45,19 @@ const GroupsPage = () => {
   const actionCommand = {
     key: 'command',
     title: t('deviceCommand'),
+    tooltip: t('commandSend')
+      .concat(' ')
+      .concat(t('reportGroup'))
+      .concat(' ')
```

Comment:
Concatenating strings is not the right solution.

Reply:
How should they be resolved?

Adding new strings to the translation file? There are a lot of dynamically strings that will have to be added..
"Remove Group", "Remove Driver", "Remove Geofence" etc

Reply:
Well, I don't think you need to duplicate information. If it's a command button, it can just say "command" or "send command". We're already on the group screen in this case, so you don't need to add that information.

Reply:
I've updated to remove the type of item from the tooltips (i.e. "Remove" instead of "Remove Group").

I've left in a couple of more specific custom translations